TLM/TLM/Manager/Impl/ExtVehicleManager.cs

```diff
@@ -1041,6 +1041,54 @@ public class ExtVehicleManager
             }
         }
 
+        /// <summary>
+        /// Converts game VehicleInfo.VehicleType to closest TMPE.API.Traffic.Enums.ExtVehicleType
+        /// </summary>
+        /// <param name="vehicleType"></param>
+        /// <returns></returns>
+        public static ExtVehicleType ConvertToExtVehicleType(VehicleInfo.VehicleType vehicleType) {
+            ExtVehicleType extVehicleType = ExtVehicleType.None;
+            if ((vehicleType & VehicleInfo.VehicleType.Car) != VehicleInfo.VehicleType.None) {
+                extVehicleType = ExtVehicleType.Bus;
+            }
```

Comment:
This will always freak me out.

Reply:
If we treat cars as buses, and buses are set to ignore lane arrows, does that mean cars will also ignore lane arrows? Also, why not set to `ExtVehicleType.PassengerCar`?

Reply:
This code is for all **unsupported** vehicle types and it's not perfect. There is no vehicle reference so no way to check AI